manifests/istio-control/istio-discovery/files/injection-template.yaml

```diff
@@ -236,6 +236,12 @@ template: |
     - name: ISTIO_META_MESH_ID
       value: "{{ .Values.global.trustDomain }}"
     {{- end }}
+    {{- if .Values.global.proxy.env }}
```

Comment:
this is what `.ProxyConfig.ProxyMetadata` is for

Reply:
@howardjohn  So how do I use it as end user? 
If I run the command below:
```
istioctl manifest generate --set meshConfig.defaultConfig.proxyMetadata.SOME_ENV_KEY=SOME_ENV_VALUE
```
will I get the SOME_ENV_KEY : SOME_ENV_VALUE passing as env to all istio-proxy containers, including egressGateway and ingressGateway?